pkg/nodeagent/node_agent.go

```diff
@@ -89,3 +113,31 @@ func GetPodSpec(ctx context.Context, kubeClient kubernetes.Interface, namespace
 
 	return &ds.Spec.Template.Spec, nil
 }
+
+func GetConfigs(ctx context.Context, namespace string, kubeClient kubernetes.Interface) (*Configs, error) {
+	cm, err := kubeClient.CoreV1().ConfigMaps(namespace).Get(ctx, configName, metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil, nil
+		} else {
+			return nil, errors.Wrapf(err, "error to get node agent configs %s", configName)
+		}
+	}
+
+	if cm.Data == nil {
+		return nil, errors.Errorf("data is not available in config map %s", configName)
+	}
+
+	jsonString, exist := cm.Data[dataPathConConfigName]
```

Comment:
So the current configmap data is as follows:
```
data-path-concurrency: {"k":"v"}
```

When we add new configs to the node agent, the configmap becomes:
```
data-path-concurrency: {"k":"v"}
new-config: {"k":"v"}
```

How about putting all configs into a JSON like this?
```
config : {"data-path-concurrency":{"k":"v"}, "new-config": {"k":"v"}}
```
In this way, we don't need to change the code in this function when we adding new configs and it is also easier for users to create the configmap with the command `kubectl create cm node-agent-configs -n velero --from-file=config=<json file name>`.

Reply:
Good idea, I have done the changes.